ecommerce/api.py

```diff
@@ -859,7 +859,9 @@ def create_unfulfilled_order(validated_basket, affiliate_id=None, **kwargs):
         )
 
         try:
-            tax_rate_info = TaxRate.objects.get(country_code=country_code)
+            tax_rate_info = TaxRate.objects.filter(active=True).get(
+                country_code=country_code
+            )
```

Comment:
We need to do the same for the BasketSerializer. Currently, in basket serializer.
`
    return TaxRate.objects.get(country_code=country_code).to_dict()
`

It shows tax on the checkout page but not in Cybersource.
<img width="1378" alt="Screen Shot 2023-09-30 at 10 23 41 AM" src="https://github.com/mitodl/mitxpro/assets/52656433/b40d637a-d542-44df-b852-35fe533f7980">

Reply:
Should be fixed in https://github.com/mitodl/mitxpro/pull/2775/commits/8521d01237a990d55dc4528e989bcaf7ed520e08